src/blog/2022/011-form-js-0.10.0.md

```diff
@@ -0,0 +1,141 @@
+</p>
+
+<!-- continue -->
+
+
+The release introduces new form components such as ‘image view’, ‘multiline text’, and ‘date time’, in order to support all basic data types and form needs. 
+It also adds the ability to add a prefix and/or suffix adornment to selected components.
+Form-js now also supports decimal numbers.
+
+Another highlight is that you are now able to hide components based on conditions and get a live preview of your form with the ability to validate it.
+
+The whole release is dedicated to helping you to build more flexible forms with awesome usability.
+Try out the new features on https://demo.bpmn.io/form/s/start.
+
+## Image View
+
+The new image view components lets you display images based on a link, data URI or variables.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-image-view.gif">
+  </a>
+
+  <p class="caption">
+    Display an image from any source using the image view component.
+  </p>
+</div>
+
+## Text View
+
+The text view component now supports FEEL expressions to display additional context via form variables and other form fields. Format your text using markdown or basic HTML.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-text-view.gif">
+  </a>
+
+  <p class="caption">
+    You can reference the value of other form components to display in the text view component.
+  </p>
+</div>
+
+## Date and Time
+
+The date and time component allows your users to display and select a date, time, or both. It is fully accessible from the keyboard.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-date-time.gif">
+  </a>
+
+  <p class="caption">
+    The new date and time picker is designed for maximum convenience, whether accessed with the mouse or keyboard.
+  </p>
+</div>
+
+## Prefix and Suffix Adornments
+
+With the new prefix and suffix adornments you are able to enrich your components with additional information, and provide guidance for your users.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-adornments.gif">
+  </a>
+
+  <p class="caption">
+    Adornments provide guidance for users in a more convenient way than labels or hints.
+  </p>
+</div>
+
+## Decimal Numbers
+
+The number component now supports decimals. You are able to define the maximum allowed decimal digit and also the increment.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-decimals.gif">
+  </a>
+
+  <p class="caption">
+    Decimal number inputs are crucial for many form applications.
+  </p>
+</div>
+
+## Hiding Components Based on FEEL Expressions
+
+With this feature you are able to hide components based on variables and conditions.
```

Comment:
May be worth making a mention that we're generally making a big move towards FEEL in the future to power up many of our properties.

Reply:
:+1: for this. 

You can get inspired by this statement: 

> Acknowledging FEEL as the script language of choice in current and upcoming OMG standards, we put significant work into FEEL editor tooling. One important building block is [feel-editor](https://github.com/bpmn-io/feel-editor), an embeddable editor for DMN FEEL expressions built on top of [CodeMirror](https://codemirror.net/). It includes syntax highlighting, validation, and auto-completion.

(From https://bpmn.io/blog/posts/2022-year-in-review.html).

Reply:
=> We embrace FEEL as the one script language across our tools.

Reply:
@nikku  expression language, not script language, right?

Reply:
Yes, sorry :).